phase4/SimpleCalculator.BusinessTest/OperatorBusiness/Operators/DivisionOperatorTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using FluentAssertions;
+using SimpleCalculator.Business.OperatorBusiness.Operators;
+
+namespace SimpleCalculator.Test.OperatorBusiness.Operators;
+
+public class DivisionOperatorTests
+{
+    [Theory]
+    [InlineData(1, 2, 1/2)]
+    [InlineData(3, 4, 3/4)]
+    [InlineData(-45, 111, -45/111)]
+    [InlineData(int.MaxValue, 1, int.MaxValue)]
+    public void Test_DivideCalculate(int first, int second, int expected)
+    {
```

Comment:
first and second are not appropriate names. Choose something that is self explanatory.